File: features/discovery/wishlist_common/src/main/java/com/tokopedia/wishlist/common/data/source/cloud/model/ProductIsWishlistedResponse.java
package com.tokopedia.wishlist.common.data.source.cloud.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ProductIsWishlistedResponse {

    @SerializedName("ProductWishlistQuery")
    @Expose
    private boolean isWishlisted;

    public boolean isWishlisted() {
        return isWishlisted;
    }

    public void setWishlisted(boolean wishlisted) {
        isWishlisted = wishlisted;
    }
}

File: features/transaction/wishlist_common/src/main/java/com/tokopedia/wishlistcommon/data/response/ProductIsWishlistedResponse.kt
package com.tokopedia.wishlistcommon.data.response

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ProductIsWishlistedResponse {
    @SerializedName("ProductWishlistQuery")
    @Expose
    var isWishlisted = false
}